emojirades/commands/scorekeeper_commands/history_command.py

```diff
@@ -27,7 +27,7 @@ def execute(self):
         self.logger.debug(f"Printing history: {history}")
 
         history_log = [
-            f"{index}. <@{event['user_id']}> > '{event['operation']}'"
+            f"{index}. <{self.slack.pretty_name(event['user_id'])}> > '{event['operation']}'"
```

Comment:
OK take 2 is closer, this would result in: `1. <Michael Robbins> > '++'`

We just need to remove those surrounding `<` and `>`!

Reply:
Ah bugger yep.